api/graphql/schema.graphql

```diff
@@ -684,7 +684,7 @@ type Query {
     )
   # countApplicants returns the number of candidates matching its filters, and requires no special permissions.
   countApplicants(where: ApplicantFilterInput): Int!
-    @count(model: "User", scopes: ["inITPublishingGroup"])
```

Comment:
I think I'd rather see two scopes used here (inItPublishingGroup, and available) instead of one scope which just combines them.